html/components/_spinners.scss

```diff
@@ -35,3 +35,9 @@
   border-left-color: $sprk-spinner-color-dark;
   border-bottom-color: $sprk-spinner-color-dark;
 }
+
+.sprk-c-Spinner--red {
```

Comment:
I left a comment about this --red up above. Do we need both --dark and --red?

Reply:
@afebbraro Left a comment about this above.